features/FEATURE_LWIP/lwip-interface/lwip_stack.c

```diff
@@ -283,6 +283,9 @@ static void add_dns_addr_to_dns_list_index(const u8_t addr_type, const u8_t inde
 
 static int get_ip_addr_type(const ip_addr_t *ip_addr)
 {
+#if LWIP_IPV6 && LWIP_IPV4
```

Comment:
This isn't correct - if it's going to be done like this, it needs to be at the end - the return remains in the same place, but with the added `#if` to shut the compiler up.

Reply:
@kjbracey-arm Updated in last commit.